databases/week1/db-week1-hw.sql

```diff
@@ -0,0 +1,58 @@
+1. Find out how many tasks are in the task table
+SELECT COUNT(*) FROM task;
+
```

Comment:
Remove empty lines of code for better practice.
Lines nr.3,6,7,16.17 and soon. This will keep your code short, precise & clean.